cmd/dagger/run.go

```diff
@@ -46,6 +46,7 @@ DAGGER_SESSION_PORT and DAGGER_SESSION_TOKEN will be convieniently injected auto
 }
 
 var waitDelay time.Duration
+var useShinyNewTUI bool = os.Getenv("_EXPERIMENTAL_DAGGER_TUI") == "1"
```

Comment:
Any value other than `""` should enable it IMO.

Reply:
You're right - I blame Copilot. :)